services/cosmvm/src/node-setup/start_node.star

```diff
@@ -0,0 +1,116 @@
+START_COSMOS = "github.com/hugobyte/dive/services/cosmvm/start-cosmos-0.sh"
+START_COSMOS_1 = "github.com/hugobyte/dive/services/cosmvm/start-cosmos-1.sh"
+DEFAULT_CONTRACT_PATH = "github.com/hugobyte/dive/services/cosmvm/static_files/contracts"
+SERVICE_NAME = "cosmos"
+SERVICE_NAME_1 = "cosmos1"
+IMAGE = "archwaynetwork/archwayd:constantine"
+PATH = "/start-scripts/"
+CONTRACT_PATH = "/root/contracts/"
+CHAIN_ID = "my-chain"
+CHAIN_ID_1 = "chain-1"
+PUBLIC_IP_ADDRESS = "127.0.0.1"
+
+def start_cosmos_node(plan,args):
+
+    plan.print("Launching " +SERVICE_NAME+  " deployment service")
+
+    plan.upload_files(src=START_COSMOS, name="start-script")
+    plan.upload_files(src=DEFAULT_CONTRACT_PATH, name="contract")
+
+    cosmwasm_node_config = ServiceConfig(
+        image=IMAGE,
+        files={ 
+            PATH: "start-script",
+            CONTRACT_PATH: "contract",
+        },
+        ports={
+            "grpc" : PortSpec(number=9090),
+            "http" : PortSpec(number=9091),
+            "tcp"  : PortSpec(number=26656),
+            "rpc"  : PortSpec(number=26657)
+        },
+        public_ports={
+            "grpc" : PortSpec(number=9090),
+            "http" : PortSpec(number=9091),
+            "tcp"  : PortSpec(number=26656),
+            "rpc"  : PortSpec(number=4564)
+        },
```

Comment:
Ports should not be added directly, Either it should be taken from Constants or User inputs. Basically it should be configurable

Reply:
updated